Update README.md (#897)
livebook-dev/livebook: README.md

  * Code notebooks with Markdown support and Elixir cells where code is evaluated on demand.

>   * Shareable: notebooks are stored in the `.livemd` format, which is a subset of Markdown with annotations, visualizations & diagrams with [Mermaid](https://mermaid-js.github.io/mermaid) and [KaTex](https://katex.org/) for mathematical formulas. note This means your notebooks can be saved, easily shared, and play well with version control.

Reviewer: ```suggestion
  * Shareable: notebooks are stored in the `.livemd` format, which is a subset of Markdown with support for diagrams via [Mermaid](https://mermaid-js.github.io/mermaid) and for mathematical formulas via [KaTex](https://katex.org/). `.livemd` files can be easily shared and play well with version control.
```
